index.html

```diff
@@ -159,15 +159,18 @@ <h4 class="">Skills</h4>
         </section>
         
         
-    </footer>
-        <h2 class="head-contact">
-        <form action="" class="form-contact">
-            <input type="text" placeholder="  Full name" name="name" id="name">
-            <input type="email" placeholder="  Email address" name="email" id="email">
-            <textarea name="message" id="message" cols="30" rows="10">  Enter text here</textarea>
+    <footer class="ContacMe">
+        <p class="head-contact">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form action="https://formspree.io/f/xjvznrod" method="POST" class="form-contact">
+            <input type="text" placeholder="  Full name" name="name" id="name" required maxlength="30">
+            <input type="email" placeholder="  Email address" name="email" id="email" required>
+            <textarea name="message" id="message" cols="30" rows="10" required maxlength="500">  Enter text here</textarea>
```

Comment:
- [ ] [OPTIONAL] Good job so far! The `Enter text here` text should paste in the `placeholder=" "` attribute If you implement this suggestion it would be better 👍🏼